src/Artemis.VisualScripting/Nodes/Text/StringLengthNode.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Artemis.Core;
+
+namespace Artemis.VisualScripting.Nodes.Text;
+
+[Node("String Length", "Checks whether the first input is contained in the second input.", 
+    "Text", InputType = typeof(string), OutputType = typeof(Numeric))]
+public class StringLengthNode : Node
+{
+    public StringLengthNode()
+        : base("String Length", "Returns string length.")
+    {
+        Input1 = CreateInputPin<string>();
+        Result = CreateOutputPin<Numeric>();
+    }
+
+    public InputPin<string> Input1 { get; }
+
+    public OutputPin<Numeric> Result { get; }
+
+    public override void Evaluate()
+    {
+        Result.Value = new Numeric((Input1.Value ?? "").Length);
```

Comment:
You can avoid allocating and counting the length of a string here when it is `null`

```cs
Result.Value = Input1.Value == null ? new Numeric(0) : new Numeric(Input1.Value.Length);
```

Reply:
Sorry, I'm originally Java developer and this wouldn't allocate in Java 😄